app/views/idv/_acuant_sdk_document_capture_form.html.erb

```diff
@@ -1,66 +1,64 @@
-<% if Figaro.env.acuant_sdk_document_capture_enabled == 'true' %>
-  <% content_for :meta_tags do %>
-    <meta
-      name='acuant-sdk-initialization-endpoint'
-      content='<%= Figaro.env.acuant_sdk_initialization_endpoint %>'
-    >
-    <meta
-      name='acuant-sdk-initialization-creds'
-      content='<%= Figaro.env.acuant_sdk_initialization_creds %>'
-    >
-  <% end %>
+<% content_for :meta_tags do %>
+  <meta
+    name='acuant-sdk-initialization-endpoint'
+    content='<%= Figaro.env.acuant_sdk_initialization_endpoint %>'
+  >
```

Comment:
I know this was picked up and moved, but maybe this is a good use case for some rails helpers? Similar to `_acuant_sdk_selfie_capture_form.html.erb`

```erb
<%= tag :meta, name: 'acuant-sdk-initialization-endpoint', content: Figaro.env.acuant_sdk_initialization_endpoint %>
<%= tag :meta, name: 'acuant-sdk-initialization-creds', content: Figaro.env.acuant_sdk_initialization_creds %>
```

Reply:
> I know this was picked up and moved, but maybe this is a good use case for some rails helpers? Similar to `_acuant_sdk_selfie_capture_form.html.erb`
> 
> ```
> <%= tag :meta, name: 'acuant-sdk-initialization-endpoint', content: Figaro.env.acuant_sdk_initialization_endpoint %>
> <%= tag :meta, name: 'acuant-sdk-initialization-creds', content: Figaro.env.acuant_sdk_initialization_creds %>
> ```

Fine to make the change here 👍 

There was another instance as well.

Updated both in 55916c88016914b7e4cc900a1787d7efd6978166.